server.php

```diff
@@ -87,23 +87,8 @@ function out($message, bool $is_error=false, string $type=ErrorType::NONE){
 $db = new DatabaseManager();
 
 // Selecto kursus no datubāzes un ieliek tos masīvā
-if($_POST['action'] == "fetch_courses"){
-
-    $sql = "SELECT * FROM courses";
-    $stmt = $db->CONN->prepare($sql);
-
-    $stmt->execute();
-
-    $courses_arr = array();
-
-    while($row = $stmt->fetch(PDO::FETCH_ASSOC)) {
-        $id = $row['id'];
-        $course = $row['course'];
-        $courses_arr[] = array("id"=>$id, "course"=>$course);
-    }
-
-    echo json_encode($courses_arr);
-
+if($_POST['action'] == "get_courses"){
+    $db->get_courses();
```

Comment:
`DatabaseManager` klase ir domāta tikai datu apstrādei nevis izvadei. Šeit labāk `server.php` failā to `echo json_encode` darīt, jo `server.php` failā arī ir definēts `json response` headeris pašā augšā.

So - `DatabaseManager` tikai atdod rezultātus un `server.php` tos izvada.

Reply:
> `DatabaseManager` klase ir domāta tikai datu apstrādei nevis izvadei. Šeit labāk `server.php` failā to `echo json_encode` darīt, jo `server.php` failā arī ir definēts `json response` headeris pašā augšā.
> 
> So - `DatabaseManager` tikai atdod rezultātus un `server.php` tos izvada.

Aaa, skaidrs. Tad `DatabaseManager` apstrādā un atdod rezultātus un `server.php` tos izvada.